rubocop-discourse.gemspec

```diff
@@ -2,7 +2,7 @@
 
 Gem::Specification.new do |s|
   s.name        = "rubocop-discourse"
-  s.version     = "2.5.0"
+  s.version     = "2.6.0"
```

Comment:
I wonder if maybe we should start using semver here, and just bump the major version whenever we add a new rule?

Reply:
I wasn’t sure which number to bump to be honest.
Enabling a new rule should bump the major version you think? We’re going to be at version 10 very quickly this way :sweat_smile:

Reply:
I'm not afraid of large numbers 😄 I'm fine with us hitting rubocop-discourse@200.0.0 😁

It makes sense to bump the major version, since every new rule we add/enable is potentially a (lint-)breaking change, that possibly requires manual changes.

Reply:
I would think consolidating the changes in a single major release will be better. While we’re testing out the changes, we can always pin dev to the main branch.

Reply:
Indeed that makes sense :+1: Let’s go for version 3 then!

Reply:
Ah sorry @tgxworld, didn’t see your reply before replying myself. If we’re fine with tracking the main branch from core, then indeed it would be simpler while we’re testing the various rules :ok_hand: